### ISSUE TITLE
Revision: Figure 3 -- New Figure 4

### ISSUE THREAD

**Issue by jaybee84** [jaybee84/ml-in-rd#181]
Please take a look at this figure from the point of view of an audience less familiar with ML. What are some of the points that are confusing?

**Comment by jaybee84** [comment]
This one overall was hard to follow. Might need to revamp this figure completely with a fresh perspective.

**Comment by dvenprasad** [comment]
Current figure for reference:
![Screen Shot 2021-08-13 at 3 45 05 PM](https://user-images.githubusercontent.com/15315514/129410908-7cbdf41d-391a-4394-9bc5-d5f5d8580edd.png)

**Comment by jaclyn-taroni** [comment]
Going to drop my brain dump here for discussion next time we meet

![image](https://user-images.githubusercontent.com/19534205/148550423-f78eccf7-1fe0-4f47-b71c-0dea7b9536b4.jpeg)

**Comment by jaybee84** [comment]
@dvenprasad Can you please add the new figure to the repo?
